### PR TITLE
Fix download of EEZ to happen on local (login) node

### PR DESCRIPTION
**Pull request by timtroendle**
Fixes an issue introduced in #99.

## Checklist

Any checks which are not relevant to the PR can be pre-checked by the PR creator. All others should be checked by the reviewer. You can add extra checklist items here if required by the PR.

- [x] CHANGELOG updated
- [x] Minimal workflow tests pass
- [x] Tests added to cover contribution
- [x] Documentation updated
- [x] Configuration schema updated

### CODE REVIEW

**Review by brynpickering — COMMENTED** [review]
Hmm, this has never been a problem for me. Do you get errors when using job nodes for downloads or is this just your preference? I just wonder whether it's a good design choice to have downloads on the login node. Parallelising them would speed things up.

**Comment by timtroendle** [comment]
Yes, this fails in my setup as compute nodes have no internet access. I believe you said once it's possible to change that, but I'd rather not if not necessary.

It's true that downloads on compute nodes could be faster as they could be parallelised. But you can also parallelise downloads on the login node using `--local-cores = [number greater 1]`.

In any case, all download rules right now are local. So we should be consistent either way.

File: rules/shapes.smk [hunk 1]
```diff
@@ -14,6 +14,7 @@ SCHEMA_UNITS = {
 
 configfile: "config/default.yaml"
 localrules: download_raw_gadm_administrative_borders, raw_gadm_administrative_borders, download_raw_nuts_units
+localrules: download_eez
```
Comment (brynpickering) [inline rules/shapes.smk:17]: Is there a reason this is on a new line? I would find it easier to understand if the first line were to become a multi-line listing of localrules.

**Review by brynpickering — APPROVED** [review]

**Comment by brynpickering** [comment]
OK, would be good to think of a section in the documentation on how best to leverage snakemake args in Euro-Calliope (since their documentation is difficult to navigate, I find). Things like `--local-cores` could then be mentioned there.

File: rules/shapes.smk (hunk 1 above)
Comment (timtroendle) [reply rules/shapes.smk:17]: Hmm the first line is too long. There may be other ways of splitting this into several lines with just a single `localrules` call, but I am not exactly sure how.

**Comment by timtroendle** [comment]
Good idea. I've opened issue #144 for that.

**Pull request merged**